src/js/events/TasksEvents.js

```diff
@@ -0,0 +1,6 @@
+var TasksEvents = {
+  DELETE_TASK: "TASKS_EVENTS_DELETE_TASK",
+  DELETE_TASK_ERROR: "TASKS_EVENTS_DELETE_TASK_ERROR"
```

Comment:
I actually wanted to suggest this at some point: we are always "namespacing" these events, so I agree that we can keep them as concise as possible!